### PR TITLE
Support `.columns TABLE` command in CLI

### LINKED ISSUE #735: Support `.columns TABLE` command in CLI
## Goal
- add `.columns Table` command in CLI as a alias of `show columns from Table`
```sql
gluesql> CREATE TABLE Foo(id INT, name TEXT);
gluesql> .columns Foo
╭──────────────╮
│ Field   Type │
╞══════════════╡
│ id      Int  │
│ name    Text │
╰──────────────╯
```
## In Next PR
- Support showing column options
Like below, it should show informations for not only columns but also options like PK, Default.. 
```sql
+-------+-------------+------+-----+---------+-------+
| Field | Type        | Null | Key | Default | Extra |
+-------+-------------+------+-----+---------+-------+
| 1     | varchar(19) | NO   |     |         |       |
+-------+-------------+------+-----+---------+-------+
```

### PR DESCRIPTION
**Pull request by devgony**
To resolve #735
## Goal
add .columns Table command in CLI as a alias of show columns from Table
```sql
gluesql> CREATE TABLE Foo(id INT, name TEXT);
gluesql> .columns Foo
╭──────────────╮
│ Field   Type │
╞══════════════╡
│ id      Int  │
│ name    Text │
╰──────────────╯
```
```sql
gluesql> .columns wrong
[error] table not found: wrong
```
```sql
gluesql> .columns
[error] should specify table. eg: .columns TableName
```
## Todo
- [x] create CommandError::LackOfTable
- [x] parse 2nd param
- If 2nd param does not exists => throw LackOfTable
- else..
  - [x] let .columns command executes `show columns from TABLE`

### CODE REVIEW

File: cli/src/cli.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+use crate::command::CommandError;
```
Comment (ever0de) [inline cli/src/cli.rs:1]: Combine it with the import statement below

File: cli/src/command.rs [hunk 2]
```diff
@@ -6,8 +8,16 @@ pub enum Command {
     ExecuteFromFile(String),
 }
 
+#[derive(ThisError, Debug, PartialEq)]
+pub enum CommandError {
+    #[error("should specify table")]
+    LackOfTable,
+    #[error("command not supported")]
+    NotSupported,
+}
```
Comment (ever0de) [inline cli/src/command.rs:17]: Good 👍

**Review by ever0de — APPROVED** [review]
It's trivial, so I'll admit it first.

File: cli/src/command.rs (hunk 2 above)
Comment (devgony) [reply cli/src/command.rs:17]: Do you think it is proper even if we don't use the error messges?
I still doubt if there is another way to distinguish those two cases without using Error

Comment (ever0de) [reply cli/src/command.rs:17]: Since they're in different branches, you might be able to tell them apart, but what's clear is that this looks good.

**Review by panarch — APPROVED** [review]
Awesome! Adding `CommandError` looks also really nice.
Thanks a lot 👍 👍

**Pull request merged**